tutorial/consuming_packages/conanfile_py/complete_conanfile.py

```diff
@@ -27,3 +27,5 @@ def layout(self):
             self.folders.generators = os.path.join("build", "generators")
```

Comment:
We should also have the build folder defined here to "build"

Reply:
Please, don't forget to do the corresponding change here: https://github.com/conan-io/docs/pull/3429/files